### ISSUE TITLE
How to remove bounces in EZSwipeController?

### ISSUE THREAD

**Issue by gaganpsinghkrishnais** [goktugyil/EZSwipeController#19]
# Every thing is working fine in my end just need to check weather is it possible to remove bounce on swipe action for viewcontrollers.

**Comment by Esqarrouth** [comment]
Remove bounces like when you try to swipe more than you can?

**Comment by gaganpsinghkrishnais** [comment]
Greetings,

**Comment by Esqarrouth** [comment]
``` swift
pageViewController = UIPageViewController(transitionStyle: .Scroll, navigationOrientation: .Horizontal, options: 
pageViewController.setViewControllers([stackPageVC[stackStartLocation]], direction: .Forward,animated: true, completion: nil)
```

Try messing with these parameters inside the code. It was something simple like that but I don't remember what

**Comment by gaganpsinghkrishnais** [comment]
ok i’ll check it. anyways thanks for your help. 

> Thanks and Regards
> 
> Gaganpreet Singh
> Sr. iOS Developer
> 
> > D-185, Phase-8B, Industrial Area, Mohali, Punjab, INDIA  
> > P: (+91) 0172-4052865  |  M: (+91) 9815475040
> > www.krishnais.com http://www.krishnais.com/ | gaganpsingh@krishnais.com mailto:gaganpsingh@krishnais.com

**Comment by Esqarrouth** [comment]
http://stackoverflow.com/questions/tagged/ezswipecontroller

You can ask your question here if you want to. If you solved the problem please send a pull request. Thanks!

Also check this out: https://github.com/goktugyil/EZSwipeController/issues/7

**Comment by otymartin** [comment]
@gaganpsinghkrishnais did you figure this out?

**Comment by otymartin** [comment]
Solved in #38
